db/deploy/removed-hotel.sql

```diff
@@ -0,0 +1,252 @@
+-- Deploy rams-core:removed-hotel to pg
```

Comment:
We've gotten into an awkward position with the DB migrations.  This change did several things:
- moved some model definitions into the ``hotel`` plugin (https://github.com/magfest/hotel)
- added some fields to the ``attendee`` table
- removed a field from the ``attendee`` table
- added some additional fields to the ``attendee`` table in the ``hotel`` plugin

It's not clear how this kind of "changes being made across multiple plugins" can work with the way we're currently doing sqitch.  The only sensible alternative might be to need each event to have its own db migrations starting from a baseline of whenever it starts and then keeping those migrations up to date as changes are made to the core repo.  I'm not sure how else this kind of thing will work.

FWIW, that's what the last project I was on at work handled things.  There was a Django project with different Django apps which would get enabled for different customers, so each customer had its own set of migrations.

In the meantime, I created some migrations which remove the hotel-related tables from the core plugin, and create the new db columns which were added to the ``Attendee`` model.  Instead of using a migration, I'll probably need to just apply these manually on staging and production when this is applied, for the reasons given above.